builtin/game/constants.lua

```diff
@@ -0,0 +1,12 @@
+-- Minetest: builtin/constants.lua
+
+--
+-- Constants values for use with the Lua API
```

Comment:
Can you perhaps give a better explanation than "the Lua API" ? Something where people immediately know what's meant?